requests_auth/authentication.py

```diff
@@ -113,8 +116,9 @@ def __init__(self, kwargs):
         """
         redirect_uri_endpoint = kwargs.pop("redirect_uri_endpoint", None) or ""
         self.redirect_uri_port = int(kwargs.pop("redirect_uri_port", None) or 5000)
+        self.redirect_uri_domain = kwargs.pop("redirect_uri_domain", None) or "http://localhost"
```

Comment:
I don't think we'll be able to spin up a HTTPS connection in any case, no need to let the ability for client to make a mistake here, default should be localhost and HTTP scheme should be hardcoded to http if that's ok with you ?

Reply:
Colin, Agree :) I made the change and committed it for review